src/main/java/org/folio/search/cql/builders/ExactTermQueryBuilder.java

```diff
@@ -20,14 +21,16 @@ public class ExactTermQueryBuilder extends FulltextQueryBuilder {
 
   public static final String SCRIPT_TEMPLATE = "doc['%s'].size() == 0";
 
+  private static final String STRING_MODIFIER = "string";
```

Comment:
Maybe it would be better to move this into enum?

Reply:
I thought about this. But until there is only one modifier option I think it's okay to have just a constant.